Source/WebKit/Shared/ios/NativeWebTouchEventIOS.mm

```diff
@@ -70,7 +70,6 @@ static inline WebEventType webEventTypeForUIWebTouchEventType(UIWebTouchEventTyp
     }
 }
 
-#if defined UI_WEB_TOUCH_EVENT_HAS_STYLUS_DATA && UI_WEB_TOUCH_EVENT_HAS_STYLUS_DATA
```

Comment:
I don't see why in your code this needs to be exposed, so I'm assuming it's for a related/follow up reason?

Reply:
"Drive by" removing long-unneeded guards that nobody ever cleaned up.